api/stub/asks.go

```diff
@@ -19,29 +19,29 @@ func (a *Asks) Get(ctx context.Context) (*ask.Index, error) {
 			Miner:        "miner1",
 			Price:        5001,
 			MinPieceSize: 1004,
-			Timestamp:    uint64(time.Now().Unix()),
-			Expiry:       uint64(time.Now().Unix()),
+			Timestamp:    1,
+			Expiry:       100,
```

Comment:
`Timestamp` and `Expiry` are now of type `ChainEpoch`, which ultimately is an integer.
That integer refers to the height at which things happen (or will).